pdk.gemspec

```diff
@@ -36,6 +36,8 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'ffi', '~> 1.9.0'
   spec.add_runtime_dependency 'hitimes', '1.3.0'
   spec.add_runtime_dependency 'net-ssh', '~> 4.2.0'
+  spec.add_runtime_dependency 'httpclient', '~> 2.8.3'
+  spec.add_runtime_dependency 'concurrent-ruby', '~> 1.1.5'
```

Comment:
IIRC this is compiled gem. No issues with this?

Reply:
It will get compiled as part of the vanagon build process and the results cached into the packages, so it should be OK.